### PR TITLE
Quickfix

### PR DESCRIPTION
**Pull request by aschmitigal22**

### CODE REVIEW

File: src/schedule/index.ts [hunk 1]
```diff
@@ -71,6 +71,8 @@ export const schoolSpecialDates: any = {
   // Presidents' Day weekend
   '2 - 18 - 2022': Schedule.NONE,
   '2 - 21 - 2022': Schedule.NONE,
+
+  '4 - 15 - 2022': Schedule.MINIMUM,
```
Comment (mark-crichton) [inline src/schedule/index.ts:75]: Can you also add the block days next week?

**Review by mark-crichton — CHANGES_REQUESTED** [review]
Can you also add the block days next week?

File: src/schedule/index.ts (hunk 1 above)
Comment (aschmitigal22) [reply src/schedule/index.ts:75]: donezo

**Review by mark-crichton — APPROVED** [review]
Looks and works gud

**Pull request merged**